README.md

```diff
@@ -2,19 +2,18 @@
 
 [![OSS Lifecycle](https://img.shields.io/osslifecycle/honeycombio/cloudformation-integrations)](https://github.com/honeycombio/home/blob/main/honeycomb-oss-lifecycle-and-practices.md)
 
-This repository contains a collection of [CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html) templates for resources in [AWS](https://aws.amazon.com/) to send 
-observabiity data to [Honeycomb](https://www.honeycomb.io/).
+This repository contains a collection of [CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html) templates for resources in [AWS](https://aws.amazon.com/) to send observability data to [Honeycomb](https://www.honeycomb.io/).
+
+Note: [Terraform modules](https://github.com/honeycombio/terraform-aws-integrations) are also available that support AWS to Honeycomb Integrations.
 
 ## How does this work?
 
 ![AWS Integrations architecture](docs/overview.png?raw=true)
 
-Note: [Terraform modules](https://github.com/honeycombio/terraform-aws-integrations) are also available that support AWS to Honeycomb Integrations.
-
 ## Usage
 
-A [quick start template](README.md#quick-start) is available to offer an easy path to integrating your AWS environments with Honeycomb.
-It makes use of the of the per-integration templates below to offer configuring many of the integrations in a single CloudFormation stack.
+A [quick start template](README.md#quick-start) offers a streamlined path to integrating your AWS environments with Honeycomb.
+The template uses the per-integration templates below to offer configuring many of the integrations in a single CloudFormation stack.
```

Comment:
Question for Team - I'm not sure if "template" is the preferred subject of this sentence, but ideally would like to fix this for clarify.

Reply:
Yep. The question is "which template" here :) Thanks!
```suggestion
The quick start template uses the per-integration templates below to offer configuring many of the integrations in a single CloudFormation stack.
```